dev/config/pyramid_oereb.yml.mako

```diff
@@ -1413,6 +1409,9 @@ pyramid_oereb:
           transfer_code: Hinweis
           extract_code: Hint
 
+  # Option to check certificate for external WMS calls. Default and recommended setting: True
```

Comment:
Add a note that this concerns only the wms of themes using the standard and oereblex models.

Reply:
Note added